Examples/Examples/ContentView.swift

```diff
@@ -47,6 +50,24 @@ struct ContentView: View {
                         Label("Load HTML String", systemImage: "doc")
                             .labelStyle(.iconOnly)
                     }
+
+                    Button {
+                        Task {
+                            pdf = PDFDocument(data: try! await proxy.contentReader.pdf())
+                        }
+                    } label: {
+                        Label("Take PDF", systemImage: "printer")
+                            .labelStyle(.iconOnly)
+                    }
+                    .accessibilityIdentifier("take_pdf_button")
+
+                    if let pdf = pdf {
```

Comment:
nits: This could be slightly shorter.
```suggestion
                    if let pdf {
```